.github/ISSUE_TEMPLATE/specreview_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification creation/plan/release... lifecycle review
+title: \[MP Component] \[Version] Specification Review
+labels: creation,plan,progress,release
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the [MicroProfile-WG repository](https://github.com/microprofile/microprofile-wg) with the content defined as follows.
+
+- [ ] Specification name and version (two digit):
+- [ ] Add a label from one of: `creation`, `plan`, `progress`, `release`
```

Comment:
I don't think we need all of these labels. As I commented before, I would just add `MPWG-release`.

Reply:
I think the labels help identify at a glance what's being requested.  They also help with sorting all of the issues and PRs that will eventually be in this repo.  We'll probably be adding even more labels down the road...